CHANGELOG.md

```diff
@@ -7,6 +7,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- Add support for loading log-based Prometheus Rules in the Loki Ruler from workload clusters.
+
+### Changed
+
+- Load log-based Prometheus Rules in the Loki Ruler via Alloy Logs instead of Alloy Rules on management clusters.
```

Comment:
This is redundant with the first line, and I think we can keep this for the observability operator when alloy rules will be removed from there.